src/ckb-daemon/usb_linux.c

```diff
@@ -357,46 +371,61 @@ void* os_inputmain(void* context){
             /// non RGB Keyboard | !IS_RGB && !IS_MOUSE | nA | nA | hid_kb_translate()
             ///
             pthread_mutex_lock(imutex(kb));
-            // EP workaround for FWv3
-            // Corsair input comes through 0x81, but case 1 in keymap.c is used for 6KRO
-            uchar urbendpoint = ((kb->fwversion >= 0x300 || IS_V3_OVERRIDE(kb)) ? 2 : (urb->endpoint & 0xF));
-            if(IS_MOUSE(vendor, product)){
-                switch(urb->actual_length){
-                case 8:
-                case 10:
-                case 11:
-                    // HID mouse input
-                    hid_mouse_translate(kb->input.keys, &kb->input.rel_x, &kb->input.rel_y, -urbendpoint, urb->actual_length, urb->buffer, kb);
-                    break;
-                case MSG_SIZE:
-                    // Corsair mouse input
-                    corsair_mousecopy(kb->input.keys, -urbendpoint, urb->buffer);
-                    break;
-                }
-            } else if(IS_RGB(vendor, product)){
-                switch(urb->actual_length){
-                case 8:
-                    // RGB EP 1: 6KRO (BIOS mode) input
-                    hid_kb_translate(kb->input.keys, -1, urb->actual_length, urb->buffer);
-                    break;
-                case 21:
-                case 5:
-                    // RGB EP 2: NKRO (non-BIOS) input. Accept only if keyboard is inactive
-                    if(!kb->active)
-                        hid_kb_translate(kb->input.keys, -2, urb->actual_length, urb->buffer);
-                    break;
-                case MSG_SIZE:
-                    // RGB EP 3: Corsair input
-                    corsair_kbcopy(kb->input.keys, -urbendpoint, urb->buffer);
-                    break;
-                }
+
+            #ifdef DEBUG_USB_INPUT
+            char converted[urb->actual_length*3 + 1];
+            for(int i=0;i<urb->actual_length;i++)
```

Comment:
Please format for readability.

Reply:
This was a direct copy from https://github.com/ckb-next/ckb-next/blob/master/src/ckb-daemon/usb_linux.c#L154-L159